DuckDuckGo/en.lproj/Localizable.strings

```diff
@@ -37,9 +37,6 @@
 /* Paste and Go action */
 "action.title.pasteAndGo" = "Paste & Go";
 
-/* Reading List action */
-"action.title.readingList" = "Add to Reading List";
```

Comment:
Do you know if the localized versions of this string be removed automatically the next time we do a localization pass?

Reply:
Thanks @samsymons.  We think it will.  This will be a good test anyway.  But I'm going to defer updating the translations until something needs some new ones.